tester/CONTRACT_SHA

```diff
@@ -1 +1 @@
-a69c763f239b81c5eb46b0bbdef3459f764360dc
+8468675915fdb202cf4656644621f7d4ebaea0da
```

Comment:
This commit is from 2.0.0 instead of master.

https://github.com/omgnetwork/plasma-contracts/commits/master
https://github.com/omgnetwork/plasma-contracts/commits/v2.0.0
v2.0.0 is the default branch for plasma-contracts and it contains our "out of beta deployment", but we can't make snapshots for it just yet, so you need to take the master branch top commit.